README.md

```diff
@@ -537,6 +537,30 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='destructure-inline-let'></a> (<a href='#* <a id='destructure-inline-let'></a> (<a href='#unnecessary-enum-arguments'>link</a>) **When destructuring an enum case or a tuple, place the `let` keyword inline, adjacent to each individual property assignment.** [![SwiftFormat: hoistPatternLet](https://img.shields.io/badge/SwiftFormat-hoistPatternLet-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#hoistPatternLet)
```

Comment:
the format of the links at the beginning is off :P

Reply:
whoops, fixed!